src/hosting/functionsProxy.ts

```diff
@@ -25,7 +25,7 @@ export interface FunctionProxyRewrite {
 export default function(
   options: FunctionsProxyOptions
 ): (r: FunctionProxyRewrite) => Promise<RequestHandler> {
-  return async (rewrite: FunctionProxyRewrite) => {
+  return (rewrite: FunctionProxyRewrite) => {
```

Comment:
nit: You may want to wrap the whole function body around `new Promise` to keep the behavior the same. Before, any errors in the function body will be returned as a rejected Promise. Now it just synchronously throws (which can be really bad), unless you wrap everything.

Reply:
Makes sense. I'll wrap it. Thanks.